tests/org.jboss.ide.eclipse.as.ui.bot.test/pom.xml

```diff
@@ -172,7 +173,7 @@
 									<outputFileName>jboss-4.0.5.GA.zip</outputFileName>
 									<md5>${jbosstools.test.jboss-as-4.0.md5}</md5>
 									<unpack>true</unpack>
-								</configuration>
```

Comment:
@robstryker what happened here? Clearly this will not build without the closing tag.